website/docs/sdks/index.md

```diff
@@ -44,62 +45,59 @@ The below table shows what features the various server-side SDKs support. Note t
-| Client registration                                                                                                                            | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     | ✅                                |
-| Basic usage metrics (yes/no)                                                                                                                   | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     | ✅                                |
-| [Impression data](../advanced/impression-data.md)                                                                                              | ⭕                     | ⭕                        | ⭕                 | ⭕                         | ⭕                     | ⭕                        | ⭕                   | ⭕                                                     | N/A                               |
-| **Category: Bootstrap (beta)**                                                                                                                 |                        |                           |                    |                            |                        |                           |                      |                                                        |                                   |
-| Bootstrap from file                                                                                                                            | ✅                     | ✅                        | ✅                 | ⭕                         | ✅                     | ⭕                        | ✅                   | ⭕                                                     | ✅                                |
-| Custom Bootstrap implementation                                                                                                                | ✅                     | ✅                        | ✅                 | ⭕                         | ✅                     | ⭕                        | ✅                   | ⭕                                                     | ✅                                |
+:::note If you see an item marked with a ❌ that you would find useful, feel free to reach out to us ([on Slack](https://slack.unleash.run/), for instance) with your use case. It may not be something we can prioritize right now, but if you'd like to contribute it back to the community, we'd love to help you build it.. :::
```

Comment:
Prettier is a bit overzealous for some formatting, so we need to make sure it doesn't break admonitions.

```suggestion
:::note 

If you see an item marked with a ❌ that you would find useful, feel free to reach out to us ([on Slack](https://slack.unleash.run/), for instance) with your use case. It may not be something we can prioritize right now, but if you'd like to contribute it back to the community, we'd love to help you build it.. 

:::
```